.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: ruby
 rvm:
-  - 2.2
-  - 2.1
+  - 2.3
```

Comment:
Do you want to update this to include more recent rubies since at this point 2.3 is EOL as well?

Reply:
I'd say 2.4 and 2.5, but open to suggestions. Not necessarily concerned with 2.6 and 2.7 since the 8.2 branch seems like legacy maintenance rather than new development?